### DISCUSSION TITLE
Qute - user tags & quarkus-web-bundler

### DISCUSSION THREAD

**Discussion by akil-rails** [quarkusio/quarkus#45246] [Q&A]
As per the [qute guide, ](https://quarkus.io/guides/qute-reference#user_tags), user defined tags are automatically registered when placed under template/tags, and this works fine.

As per the [quarkus-web-bunlder guide](https://docs.quarkiverse.io/quarkus-web-bundler/dev/advanced-guides.html#qute-components), qute components can be placed under src/main/resources/web/components/[name], this also works 

The quarkus extension for vs-code however complains for 'missing section helper' when the quarkus-web-bundler guide's conventions are followed (it works when the qute-reference convention is followed) .. Is something to be done to let vscode extension know about these?

<img width="317" alt="image" src="https://github.com/user-attachments/assets/bcef9b71-8350-4895-a578-e64b54a0cb81" />
<img width="299" alt="image" src="https://github.com/user-attachments/assets/dc7e7130-ad7f-496e-93d6-32c7d3d86cee" />

[a comment by an automated account is not shown]

**Comment by mkouba** [comment]
CC @angelozerr

**Reply by angelozerr** [reply]
@akil-rails please create an issue at https://github.com/redhat-developer/quarkus-ls by sharing your project please. I dont know if it will be easy to support it.

**Reply by akil-rails** [reply]
Okay , done, raised it https://github.com/redhat-developer/quarkus-ls/issues/1040 ..
